### ISSUE TITLE
Java MicroProfile projects do not present app.log on Windows 10

### ISSUE THREAD

**Issue by sujeilyfonseca** [eclipse/codewind#102]
**VS Code version:** 1.35.1
**Eclipse version:** 2019-06 (4.12.0)
**OS:** VM with Windows 10

**Description:**
Java MicroProfile projects do not have some logs (messages.logs, app.log and console.log).

Messages and console logs were addressed at https://github.com/eclipse/codewind/pull/68.

However, the app.log problem has yet to be resolved. This log file is in the workspace. For additional details, see the following issue: https://github.com/eclipse/codewind/issues/7

![image](https://user-images.githubusercontent.com/49531960/60190170-30f07e00-9800-11e9-894e-65caf60f0a02.png)

![](https://user-images.githubusercontent.com/49531960/60206175-eb907880-9820-11e9-90a5-112197d1f34e.png)

@jagraj

**Comment by hhellyer** [comment]
@tetchel @eharris369 - The log that is failing here is the app.log. IIRC correctly that is the standard out from the process which is also available direct from the container using the log name  `-`.
If it is going to take a long time to fix the app.log issue would it be possible to use the `-` log in the meantime?

**Comment by tetchel** [comment]
I'm not sure if they are the same thing @ssh24

**Comment by ssh24** [comment]
Yes, currently the `app.log` file is the exact same as the app container output. For liberty cases, there are other logs such as `message` and `console` but that is app specific and not part of the application container.

**Comment by tetchel** [comment]
> If it is going to take a long time to fix the app.log issue would it be possible to use the - log in the meantime?

do we know how to fix the app.log?

**Comment by hhellyer** [comment]
@tetchel - I'm not sure, Turbine is returning the app.log in the list of logs but whether the logs are in the container or in the workspace isn't returned per log file turbine reports that all of them are either in the workspace or the container.
Unfortunately Windows microprofile is the only scenario where we have container logs but only `messages.log` and `console.log` are actually in the container. `app.log` is actually in the workspace.
I put a write up here: https://github.com/eclipse/codewind/issues/7#issuecomment-516798973
